crates/shared/src/sources/balancer_v2/pool_fetching.rs

```diff
@@ -298,27 +265,4 @@ mod tests {
         assert_eq!(filtered_pools.len(), 1);
         assert_eq!(filtered_pools[0].common.id, pools[1].common.id);
     }
-
```

Comment:
This just moved to `shared::sources::balancer_v2::pools::stable` without any change.